### ISSUE TITLE
The deprecated attribute

### ISSUE THREAD

**Issue by vorner** [tokio-rs/prost#221]
Hello

I have a proto file with some `[deprecated=true]` attributes in it, like this one:

```
enum Stuff {
    DEFAULT = 0;
    OUTDATED = 1 [deprecated=true];
    NEW = 2;
}
```

From what I can tell, it doesn't propagate the attribute into the generated code. Would it be hard to place the `#[deprecated]` attribute onto the generated code?

**Comment by danburkert** [comment]
Nope, should be relatively straightforward.  Sounds like a great feature addition to `prost`.
